packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -374,6 +377,28 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
       />
     );
   }, [fileSharing?.accept, fileSharing?.multiple, fileSharing?.uploadHandler, fileUploadButtonOnChange]);
+
+  /* @conditional-compile-remove(file-sharing) */
+  const activeFileUploads = useSelector(fileUploadsSelector).files;
+
+  const SelectedSendBox = (): JSX.Element => {
```

Comment:
Should we just call it SendBox and change 
```
import {
  SendBox
} from '@internal/react-components';
```

to

```
import {
  SendBox as SimpleSendBox
} from '@internal/react-components';
```

Reply:
Oh that's a good idea, will do!